skipper.go

```diff
@@ -761,9 +762,9 @@ func Run(o Options) error {
 		auth.NewOAuthOidcAllClaims(o.OIDCSecretsFile),
 		apiusagemonitoring.NewApiUsageMonitoring(
 			o.ApiUsageMonitoringEnable,
-			o.ApiUsageMonitoringRealmKey,
-			o.ApiUsageMonitoringClientIdKeyName,
-			o.ApiUsageMonitoringDefaultClientTrackingPattern,
+			o.ApiUsageMonitoringRealmKeys,
```

Comment:
on the long term we should try to avoid compatibility breaking changes, but I see how this change makes this feature more usable :+1:

Reply:
Agree! (that's why we haven't officially released the api-usage-monitoring filter within Z)